template-parts/content-excerpt.php

```diff
@@ -27,8 +27,14 @@
 	<div class="c-article__content">
 		<?php
 
-			// Advanced Custom Fields content
-			get_template_part( 'template-parts/acf', 'flex' );
+			// Don't show post content if it's in a listing page
+			if ( is_single() ) {
+					// Advanced Custom Fields content
+					get_template_part( 'template-parts/acf', 'flex' );
```

Comment:
Do we need to call the full ACF Flex if we're just trying out output the excerpt?